config/manager/manager.yaml

```diff
@@ -54,6 +54,9 @@ spec:
             value: quay.io/konveyor/kubevirt-velero-plugin:v0.2.0
           - name: RELATED_IMAGE_MUSTGATHER
             value: registry.redhat.io/oadp/oadp-mustgather-rhel8:v1.2
+          # TODO change
```

Comment:
yeah we would need something in migtools org

Reply:
TODO decide image name to create PROW job for it

Reply:
It will be `quay.io/konveyor/oadp-non-admin`